Umbraco-Cloud/Deployment/Umbraco-Forms-on-Cloud/index.md

```diff
@@ -30,45 +32,75 @@ For more information on how to handle content transfer/restores on Umbraco Cloud
+
+Whenever a new minor version of Umbraco Forms is ready, eg. 8.x or 7.x, you will get the option to apply the upgrade to your project. When your project is elligible to receive the new version, you will see an "*Upgrade available!*" label on your Development environment.
+
+### Version specific changes
+
+In this section you can find information about version specific changes that might affect the way Umbraco Forms works on your project.
+
+#### Version 8.5.0
+
+Prior to Umbraco 8.5.0 all forms data was saved as `.json` files in the `App_Data/UmbracoForms` directory in the file system.
+
+As of Umbraco 8.5.0 you have the option to persist all Forms data directly in the database. This behaviour is default to all new sites created on Umbraco Cloud since September 2020. Was your Cloud project created before, you will need to upgrade the Umbraco Forms version as well as applying a setting in order to perform the migration of the Umbraco Forms data.
+
+In order to switch to persisting all definitions for Umbraco Forms data in the Umbraco database, follow these steps:
+
+1. Make sure all environments are upgraded to **at least Umbraco Forms version 8.5.2 and Deploy 3.5.0**
+2. Make sure your forms are in sync between all your Cloud environments
+3. Clone down you Development environment
+4. Open the configuration file `App_Plugins\UmbracoForms\UmbracoForms.config` from your local clone
+5. Locate the `StoreUmbracoFormsInDb` key in the `<settings>` section, and make sure it has the follow value:
 
-Then your forms data is handled as metadata. When you create a form, a `UDA` file will be generated. This `UDA` file contains all the metadata from the form and is very similar to the `JSON` file that is also generated when you create a new form (found in `App_Data\UmbracoForms\Data\forms`).
+    ```code
+    <setting key=“StoreUmbracoFormsInDb” value=“True” />
+    ```
 
-This means that your forms will be deployed along with the rest of your metadata and structure files, e.g. Document Types, Templates and stylesheets.
+6. Save the file
+7. Spin up your local clone and verify that everything works as expected
 
-We strongly recommend that you work with the forms on your local or Development environment, following the [left-to-right deployment model](../../Deployment).
+Now it is time to deploy the setting to your Cloud environments.
 
-You can configure your project to handle forms data as content by following these steps:
+Follow the steps outlined below **for each environment** in order for the migration to run on each of them.
+
+1. Access **KUDU**
+2. Navigate to `site/wwwroot/App_Data`
+3. Delete the `UmbracoForms` directory
+4. Push/Deploy the updated setting to the environment
+5. In order to run the migration, **restart the environment** from the Cloud portal
+6. Repeat steps 1-5 for each of your Cloud environments
```

Comment:
6. From Umbraco backoffice, queue and transfer the forms to the environment
7. Repeat steps 1-5 for each of your Cloud environments